### PR TITLE
chore(agw): remove docker container privileges in agw

### PR DESCRIPTION
**Pull request by crasu**
Signed-off-by: Christian Krämer <christian.kraemer@tngtech.com>

## Summary

Removed priviliges for all running agw containers.

## Test Plan

I made sure that a least the health check for the container works.

## Additional Information

For pipelined and mme I was unable to test the changes at all so I skipped removing the privileges  there. Added required icmp capabilities to monitord as all monitord does is monitoring subscribers via icmp. 

Is there a way to run the integration tests against these containers? -> Answer: No. See seperate PR.

### CODE REVIEW

File: lte/gateway/docker/docker-compose.yaml [hunk 1]
```diff
@@ -1,4 +1,3 @@
----
```
Comment (Neudrino) [inline lte/gateway/docker/docker-compose.yaml:1]: I wonder, that yaml_lint does not complain on this one. :thinking: But OK, if it does not...

File: lte/gateway/docker/docker-compose.yaml [hunk 2]
```diff
@@ -117,8 +116,6 @@ services:
       test: ["CMD", "nc", "-zv", "localhost", "50080"]
       timeout: "4s"
       retries: 3
-    # Needed in order to enable/disable ICMP
-    privileged: true
```
Comment (Neudrino) [inline lte/gateway/docker/docker-compose.yaml:126]: Did anything change, that this is not necessary to enable/disable ICMP?

File: lte/gateway/docker/docker-compose.yaml [hunk 3]
```diff
@@ -128,6 +125,8 @@ services:
       test: ["CMD", "nc", "-zv", "localhost", "50076"]
       timeout: "4s"
       retries: 3
+    cap_add:
+      - NET_RAW
```
Comment (Neudrino) [inline lte/gateway/docker/docker-compose.yaml:134]: What is this change for? It is not listed in the PR description. Could you comment and add it there, please?

**Comment by ajahl** [comment]
> For mobilityd you could probably verify that it's basically working using https://github.com/magma/magma/blob/3fc44481fe9196b579d2b6cc6b0092e359103335/lte/gateway/python/scripts/mobility_cli.py

Mobilityd works inside the docker container.

File: lte/gateway/docker/docker-compose.yaml (hunk 2 above)
Comment (crasu) [reply lte/gateway/docker/docker-compose.yaml:126]: The  health is checked via dockerd not via icmp

File: lte/gateway/docker/docker-compose.yaml (hunk 3 above)
Comment (crasu) [reply lte/gateway/docker/docker-compose.yaml:134]: This service monitors subscribers via icmp. It will not work without icmp.

**Review by jheidbrink — APPROVED** [review]

**Review by Neudrino — APPROVED** [review]

**Pull request merged**